data/waellerbe.json

```
@@ -1,11 +1,18 @@
 {
   "name": "William A Ellerbe",
-  "bio": "Prompt Engineering Consultant with a passion for Open Source projects",
-  "links": [
+  "bio": "Open Source | Prompt Engineering | Proud member of **EddieHub**",
+"links": [
 {
+      "group": Socials Primary
```

adding quotation marks and comma:
```suggestion
      "group": "Socials Primary",
```